web-client/src/theme/variables.scss

```diff
@@ -32,7 +32,7 @@
   // UI Kit: Primary 2 - Portland Orange
   --ion-color-primary: #ff642e;
   --ion-color-primary-rgb: 255, 100, 46;
-  --ion-color-primary-contrast: #000000;
+  --ion-color-primary-contrast: #ffffff;
   --ion-color-primary-contrast-rgb: 0, 0, 0;
```

Comment:
(The `contrast` and `contrast_rgb` variations should match.)

Reply:
Done! Thanks